3ds Max/Max2Babylon/OnPostBuild.bat

```diff
@@ -5,7 +5,7 @@ SETLOCAL enabledelayedexpansion
 SET max_version=%2
 SET max_location=!ADSK_3DSMAX_x64_%max_version%!
 
-SET source_dir="%~dp0assemblies\%max_version%\"
+SET source_dir="%~dp0%max_version%\bin\%1"
```

Comment:
also, how are variables like %~dp0 and %~dp0assemblies defined? asking to learn, as I'm totally unfamiliar with this convention? :+1:

Reply:
Try running "help call" or "call /?" in a command prompt. That will explain what this syntax means. Specifically, in this case, `%~dp0` just means the path of the currently running batch script.